src/components/NewTodo/NewTodo.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import './NewTodo.css';
+import { NewTodoType } from '../../types';
+
+export class NewTodo extends React.Component {
+  state = {
+    title: '',
+    user: '',
+    titleError: false,
+    userError: false,
+  }
+
+  handleChange = (e) => {
+    const { name, value } = e.target;
+
+    this.setState({
+      [name]: value,
+      titleError: false,
+      userError: false,
+    });
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+
+    const { title, user } = this.state;
+
+    if (!user) {
+      this.setState({
+        userError: true,
+      });
+
+      return;
+    }
+
+    if (!title.trim()) {
+      this.setState({
+        titleError: true,
+      });
+
+      return;
+    }
+
+    this.props.addTodos(title, user);
```

Comment:
```suggestion
    addTodos(title, user);
```
делай вверху компонента деструктуризацию `props`